### ISSUE TITLE
[4.0] Login Page usage of icon-font

### ISSUE THREAD

**Issue by unknown-sudo** [joomla/joomla-cms#26314]
### What needs to be fixed
The login page uses an icon font for external links which can be difficult for usability since some screenreaders might read the content of the pseudo element and this can lead to some confusion. This can be significantly improved in terms of usability and performance by including svgs. SVGs have the advantage that they can be integrated accessible. Furthermore, you can create an icon sprite if you use this symbol so often. In general, this problem can be seen as a general problem for using icon fonts in the backend.

Reference:
https://css-tricks.com/svg-sprites-use-better-icon-fonts/
https://www.24a11y.com/2018/accessible-svg-icons-with-inline-sprites/

The advantage of this procedure is:
- that the SVG sprites can be easily colored with CSS.
- that the SVGs can be made barrier-free.

**Comment by brianteeman** [comment]
There is no accessibility issue here
![image](https://user-images.githubusercontent.com/1296369/64922282-83b95080-d7c5-11e9-8118-901d6371e1eb.png)

**Comment by unknown-sudo** [comment]
Please do not trust automated accessibility tools. Just because your browser doesn't report it as bad doesn't mean it's good. 

https://www.matuzo.at/blog/building-the-most-inaccessible-site-possible-with-a-perfect-lighthouse-score/

**Comment by brianteeman** [comment]
That is not an automated testing tool. Prove that it is an issue

**Comment by unknown-sudo** [comment]
If we think that Joomla 4 has to last a few years and we use an old technology like icon fonts, this deal is already a problem in its own right in my eyes.

I already mentioned in my links above that using svgs (in sprites) instead of icon fonts has a lot of advantages. 

References:
https://cloudfour.com/thinks/seriously-dont-use-icon-fonts/
https://www.24a11y.com/2017/svg-icon-fonts-accessibility-case-study/

Docs to use FontAwesome with sprites 
https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility#svg-sprites-semantic

**Comment by brianteeman** [comment]
Feel free to submit a pull request to replace all uses of the icon font with svg. I will be happy to test it. But please don't say that just because its a font icon it is not accessible without providing evidence of that in Joomla

**Comment by unknown-sudo** [comment]
For a PR I need the best possible list of icons used in the backend and their meaning. I could then provide the sprite and the code to use in the appropriate places. But the PR should be done by someone who knows the backend template code very well and knows where to put the corresponding icons. Do we want to work together here?

**Comment by brianteeman** [comment]
No I don't as I do not believe that using a sprite is the best approach for an extendable piece of software like joomla where extensions need to have the ability to chose an icon

**Comment by unknown-sudo** [comment]
Okay, well, I hope somebody comes forward to work with me on this. We should have the goal that Joomla fulfills the advertised accessibility and is also professionally and modern programmed. I see here an enormous potential for improvement for both topics. Using icon-fonts is absolutely outdated. Sprites can of course also be used by third party extensions. 
Either way it should be implemented semantically correct.

**Comment by N6REJ** [comment]
@unknown-sudo if you can show me a case where the accessibility is an issue I will address that particular case.  Just because svg is the new trend doesn't make it better.  While there are many things I'm not crazy about with the current icon system, none of it has to do with the viability of font/svg.
Your claim says that fontawesome doesn't know what they are doing.
